packages/commcare/src/Utils.js

```diff
@@ -0,0 +1,43 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  makeBasicAuthHeader,
+  logResponse,
+} from '@openfn/language-common/util';
+
+export const prepareNextState = (state, response, callback) => {
+  const { body, ...responseWithoutBody } = response;
+  const nextState = {
+    ...composeNextState(state, response.body),
+    response: responseWithoutBody,
+  };
+
+  return callback(nextState);
+};
+
+export function request(state, method, path, data, params, header, authType) {
```

Comment:
This signature is getting very long! Instead of declaring all these parameters, I would just declare one: an object.

Basically just wrap the parameters up into an object, like this:
```
export function request({ state, method, path, data, params, header, authType })
```